easybuild/tools/config.py

```diff
@@ -865,6 +867,46 @@ def log_path(ec=None):
     return log_file_format(return_directory=True, ec=ec, date=date, timestamp=timestamp)
 
 
+def get_failed_installs_build_dirs_path(ec):
+    """
+    Return the 'failed_installs_build_dirs_path',
+    the location where build directories are copied if installation failed
+
+    :param ec:  dict-like value with 'name' and 'version' keys defined
+    """
+    failed_installs_build_dirs_path = ConfigurationVariables()['failed_installs_build_dirs_path']
+
+    if not failed_installs_build_dirs_path:
+        return None
+
+    try:
+        name, version = ec['name'], ec['version']
+    except KeyError:
+        raise EasyBuildError("The 'name' and 'version' keys are required.")
+
+    return os.path.join(failed_installs_build_dirs_path, name + '-' + version)
```

Comment:
```suggestion
    return os.path.join(failed_installs_build_dirs_path, f'{name}-{version}')
```
This way it won't fail for `None` which seemingly is possible according to previous discussions